pkg/handlers/v1/testdata/ec2.deleted.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
test message which is an exact copy of the config message that was received (and failed) in prod